### PR TITLE
Tokens: generate commonJS constant file

### PR DESCRIPTION
**Pull request by AlbertCarreras**
Tokens: generate commonJS constant file
Screenshot by Dropbox Capture

To prevent this issue in Pinboard 
image

the issue is because the file is using export vs the cjs module.exports

In Pinboard, we ignore _node_modules_ by default for Jest

i.e. for gestalt we ship both ecjs and esm equivalents, from a consistency perspective that would make more sense to have both (but if we have to pick one we should stick with esm. Its more ideal for treeshaking on the pinboard side
since you can't treeshake cjs)

### CODE REVIEW

File: packages/gestalt-design-tokens/build.js [hunk 1]
```diff
@@ -150,6 +150,39 @@ StyleDictionary.registerFormat({
...
+        // `getReferences()` methods. `usesReference()` will return true if
+        // the value has a reference in it. `getReferences()` will return
+        // an array of references to the whole tokens so that you can access their
+        // names or any other attributes.
+        if (dictionary.usesReference(token.original.value)) {
+          // Note: make sure to use `token.original.value` because
+          // `token.value` is already resolved at this point.
+          const refs = dictionary.getReferences(token.original.value);
+          refs.forEach((ref) => {
+            value = value.replace(ref.value, () => `${ref.name}`);
```
Comment (diyorbek) [inline packages/gestalt-design-tokens/build.js:169]: Let's make it simpler
```suggestion
            value = value.replace(ref.value, ref.name);
```

File: packages/gestalt-design-tokens/build.js [hunk 2]
```diff
@@ -150,6 +150,39 @@ StyleDictionary.registerFormat({
...
+          });
+        }
+        return `  TOKEN_${token.path
+          .join('_')
+          .toUpperCase()
+          .replace('-', '_')}: 'var(--${token.path.join('-')})',`;
+      })
+      .join(`\n`);
+
+    return `${fileHeader({
```
Comment (diyorbek) [inline packages/gestalt-design-tokens/build.js:179]: can we move this part to a variable? to make it more readable.

**Review by diyorbek — APPROVED** [review]

File: packages/gestalt-design-tokens/build.js (hunk 1 above)
Comment (AlbertCarreras) [reply packages/gestalt-design-tokens/build.js]: DONE

File: packages/gestalt-design-tokens/build.js (hunk 2 above)
Comment (AlbertCarreras) [reply packages/gestalt-design-tokens/build.js]: DONE

**Pull request merged**